app/models/language_model.rb

```diff
@@ -4,12 +4,14 @@ class LanguageModel < ApplicationRecord
   BEST_CLAUDE = "claude-best"
   BEST_GROQ = "groq-best"
 
+  # TODO: infer these from models.yaml
   BEST_MODELS = {
     BEST_GPT => "gpt-4o-2024-08-06",
     BEST_CLAUDE => "claude-3-5-sonnet-20240620",
     BEST_GROQ => "llama3-70b-8192",
   }
 
+  # TODO: what are these for?
```

Comment:
The app now tracks your usage $. Currently, when you click "..." on a conversation you see this:

<img width="373" alt="image" src="https://github.com/user-attachments/assets/6344a199-fb23-460b-90d3-fc4cfb71edba">

Soon I want to add it so that when you click on your profile pic (button left) then it shows something equivalent for the month-to-date across all conversations.

But... (note to self) these two things should probably be moved to a separate concern of their own. They're not so important as to be so prominent in this model.